### ISSUE TITLE
Colab notebook not working

### ISSUE THREAD

**Issue by vaclavhanzl** [aqlaboratory/openfold#286]
Colab notebook pointed to from the main page is not working for multiple reasons, throwing quite confusing error messages.
I tried to fix it by PR #285. You can test the fixed notebook [here](https://colab.research.google.com/github/vaclavhanzl/openfold/blob/fix-colab-notebook/notebooks/OpenFold.ipynb). Testers welcome!

**Comment by vaclavhanzl** [comment]
Colab notebook was fixes by merging my #285 but at the same day broken again by also merging #287 :)
Now Colab fails on missing `modelcif` dependency, This should be easy to fix.

**Comment by burunduk57** [comment]
> Colab notebook was fixed by merging my #285 but at the same day broken again by also merging #287 :) Now Colab fails on missing `modelcif` dependency, This should be easy to fix.

I get missing  `modelcif` problem

**Comment by vaclavhanzl** [comment]
> I get missing `modelcif` problem

And you would get more problems down the road (changed parameters of functions and a very cryptic failure due to Python version discrepancy). I proposed a fix but until it gets merged, you can try this:

Open the notebook [here](https://colab.research.google.com/github/vaclavhanzl/openfold/blob/colab-add-dep-fix-python-version/notebooks/OpenFold.ipynb) and edit the Install OpenFold cell (change GIT_REPO and add branch checkout):
```
#GIT_REPO='https://github.com/aqlaboratory/openfold'
GIT_REPO='https://github.com/vaclavhanzl/openfold'
...
    %shell git clone "{GIT_REPO}" openfold 2>&1 1> /dev/null
    %shell cd openfold && git checkout colab-add-dep-fix-python-version
```

**Comment by vaclavhanzl** [comment]
@burunduk57 to make it easier for you (and for others), I merged all my fixes in [my fork of the OpenFold repo](https://github.com/vaclavhanzl/openfold/tree/vhmain) and just added one more fix (openmm version). Please try [my fixed version of the Colab notebook](https://colab.research.google.com/github/vaclavhanzl/openfold/blob/vhmain/notebooks/OpenFold.ipynb). It works for me now.

**Comment by burunduk57** [comment]
> @burunduk57 to make it easier for you (and for others), I merged all my fixes in [my fork of the OpenFold repo](https://github.com/vaclavhanzl/openfold/tree/vhmain) and just added one more fix (openmm version). Please try [my fixed version of the Colab notebook](https://colab.research.google.com/github/vaclavhanzl/openfold/blob/vhmain/notebooks/OpenFold.ipynb). It works for me now.

It works for me, many thanks!

**Comment by vaclavhanzl** [comment]
This notebook is really cat-n-mouse game :) What worked 5 days ago does not work now. My fix was finally just merged to main now, but now the notebook fails at:
```
%shell conda install -y -q -c conda-forge pdbfixer
```
I'll look into it once more, maybe there are more things to be automated (my auto-following of the python version already saved us one breakage).

**Comment by vaclavhanzl** [comment]
I fixed it in #320.

**Comment by vaclavhanzl** [comment]
Merged (thanks @gahdritz), tested, works! Closing the issue.
